app/models/ssh_key.rb

```diff
@@ -0,0 +1,66 @@
+class SshKey < ActiveRecord::Base
+  include Authorizable
```

Comment:
I'm reluctant on this one. It might be possible, but imho we try to model something completely different. There isn't any real code duplication now and the models behave differently. If we force them into a single model, it just gets harder to maintain and test this.